### PR TITLE
Add Cone Geometry Message

### PR DESCRIPTION
**Pull request by Levi-Armstrong**

### CODE REVIEW

**Comment by chapulina** [comment]
Closes #46

File: proto/ignition/msgs/conegeom.proto [hunk 1]
```diff
@@ -0,0 +1,36 @@
+/*
+ * Copyright (C) 2017 Open Source Robotics Foundation
```
Comment (iche033) [inline proto/ignition/msgs/conegeom.proto:2]: can you update the year to `2020`?

**Comment by iche033** [comment]
can you sign off your commit? We are now requiring this for DCO check to pass. You can click on [Details](https://github.com/ignitionrobotics/ign-msgs/pull/47/checks?check_run_id=637107124) beside `DCO` for more info.

File: proto/ignition/msgs/conegeom.proto (hunk 1 above)
Comment (Levi-Armstrong) [reply proto/ignition/msgs/conegeom.proto]: Updated

File: proto/ignition/msgs/conegeom.proto [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+/// \brief Information about a cone geometry
+
+import "ignition/msgs/header.proto";
+
+message ConeGeom
+{
+  /// \brief Optional header data
+  Header header  = 1;
+
+  double radius  = 2;
```
Comment (chapulina) [inline proto/ignition/msgs/conegeom.proto:34]: Could you document all fields in the new message? The unit would also be important (i.e. `meters`). Thanks!

Comment (Levi-Armstrong) [reply proto/ignition/msgs/conegeom.proto:35]: Done

**Pull request merged**